app/serializable/serializable_submission.rb

```diff
@@ -7,7 +7,7 @@ class SerializableSubmission < JSONAPI::Serializable::Resource
   belongs_to :task
   has_many :files
 
-  attributes :framework_id, :supplier_id, :task_id
+  attributes :framework_id, :supplier_id, :created_by_id, :task_id
```

Comment:
I don't think we should expose this here as there are no plans to do anything with it on the supplier-facing application, which is where this ends up.